### ISSUE TITLE
Smart blksize negotiation decreases performance massively

### ISSUE THREAD

**Issue by johann-pfefferl** [pin/tftp#51]
In my application I use a blksize of 65432 to transfer big ISO images by tftp (speed 60MB/s). With the new blksize logic the speed now drops to 10MB/s. Is it possible to introduce a boolean flag to disable this smart negotiation handling for people who know their network components and simply use the blksize value again which the client provides.

**Comment by pin** [comment]
@johann-pfefferl, sorry for the slow response and thanks for heads up! It sounds like a trivial change. Are you still interested in it?

**Comment by johann-pfefferl** [comment]
Yes I am still interested in this feature! I heavily use this tftp server in a test environment.

**Comment by pin** [comment]
I'm still contemplating a bit on the proper method name, but will land #127 soon.

**Comment by pin** [comment]
I've added `server.SetBlockSizeNegotiation(false)` method to disable MTU-based block size clamping. Benchmark shows perf improvement.
